stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/message/MessageSendingService.kt

```diff
@@ -102,12 +110,16 @@ internal class MessageSendingService(
                     }
             }
             )
+        enqueueMessage(newMessage)
+        return Result.success(newMessage)
+    }
+
+    private fun enqueueMessage(message: Message) {
```

Comment:
To be clear, maybe rename this method name as `enqueueAttachmentUploadForMessage` or something similar?